app/models/Notifications.scala

```diff
@@ -156,6 +156,14 @@ object NotificationUserPreference {
       client.set(s"""NotificationUserPreference:${ConferenceDescriptor.current().eventCode}:${webUserId}""", json)
   }
 
+  def loadAll(webUserIds: List[String]): Map[String, NotificationUserPreference] = Redis.pool.withClient { implicit client =>
+    val notifUserPrefsKeys = webUserIds.map(webUserId => s"""NotificationUserPreference:${ConferenceDescriptor.current().eventCode}:${webUserId}""")
+    val notifPrefEntries = for ((webUserId, notificationPrefJSON) <- (webUserIds zip client.mget(notifUserPrefsKeys:_*)))
```

Comment:
@fcamblor I have a bug when I call http://localhost:9000/admin/initializeWatchingProposals


```
[JedisDataException: ERR wrong number of arguments for 'mget' command]

In cfp-devoxx/app/models/Notifications.scala:161
158
159  def loadAll(webUserIds: List[String]): Map[String, NotificationUserPreference] = Redis.pool.withClient { implicit client =>
160    val notifUserPrefsKeys = webUserIds.map(webUserId => s"""NotificationUserPreference:${ConferenceDescriptor.current().eventCode}:${webUserId}""")
161    val notifPrefEntries = for ((webUserId, notificationPrefJSON) <- (webUserIds zip client.mget(notifUserPrefsKeys:_*)))
162      yield webUserId -> Json.parse(notificationPrefJSON).as[NotificationUserPreference]
163
164    notifPrefEntries.toMap
165  }
```

Reply:
Didn't you forget the case where a user didn't configure any NotificationUserPreference ?